projects/distributed-tracing/src/shared/dashboard/interaction/detail-sheet/detail-sheet-interaction-handler.service.ts

```diff
@@ -0,0 +1,31 @@
+import { PortalInjector } from '@angular/cdk/portal';
+import { Injectable, Injector } from '@angular/core';
+import { OverlayService, PopoverRef, SheetSize } from '@hypertrace/components';
+import {
+  DetailSheetInteractionContainerComponent,
+  DETAIL_SHEET_INTERACTION_MODEL
+} from './container/detail-sheet-interaction-container.component';
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
one of the dependencies of this, overlay service, is not a root injectable (because it requires an injector). So this probably shouldn't be either.

Reply:
Yes. I agree. Wanted to confirm this in PR review. I would provide the service at the module level.